pkg/chains/signing.go

```diff
@@ -111,11 +111,7 @@ func (o *ObjectSigner) Sign(ctx context.Context, tektonObj objects.TektonObject)
 	}
 
 	signers := allSigners(ctx, o.SecretPath, cfg, logger)
-
-	rekorClient, err := getRekor(cfg.Transparency.URL, logger)
-	if err != nil {
-		return err
-	}
+	var rekorClient rekorClient
```

Comment:
Is there a benefit to declaring this variable this far from when it's actually used?

It could be useful to only have to call `getRekor` once during this method, but that doesn't seem to be the case here.